.github/CODEOWNERS

```diff
@@ -109,63 +109,13 @@
 /codecov.yml @kevinlee12
 /core/templates/css/.stylelintrc @kevinlee12
 /tox.ini @kevinlee12
-/scripts/linters/__init__.py @kevinlee12 @Hudda
-/scripts/linters/codeowner_linter.py @kevinlee12 @Hudda
-/scripts/linters/css_linter.py @kevinlee12 @Hudda
-/scripts/linters/general_purpose_linter.py @kevinlee12 @Hudda
-/scripts/linters/html_linter.py @kevinlee12 @Hudda
-/scripts/linters/js_ts_linter.py @kevinlee12 @Hudda
-/scripts/linters/linter_utils.py @kevinlee12 @Hudda
-/scripts/linters/linter_utils_test.py @kevinlee12 @Hudda
-/scripts/linters/pre_commit_linter.py @kevinlee12 @Hudda
-/scripts/linters/pylint_extensions.py @kevinlee12 @Hudda
-/scripts/linters/pylint_extensions_test.py @kevinlee12 @Hudda
-/scripts/linters/python_linter.py @kevinlee12 @Hudda
-/scripts/__init__.py @kevinlee12
-/scripts/build.py @kevinlee12
-/scripts/build_test.py @kevinlee12
-/scripts/check_e2e_tests_are_captured_in_ci*.py @Showtim3 @kevinlee12
-/scripts/check_frontend_coverage.py @marianazangrossi @kevinlee12
-/scripts/check_frontend_coverage_test.py @marianazangrossi @kevinlee12
-/scripts/clean.py @kevinlee12
-/scripts/clean_test.py @kevinlee12
-/scripts/common.py @kevinlee12
-/scripts/common_test.py @kevinlee12
-/scripts/create_expression_parser.py @kevinlee12
-/scripts/create_topological_sort_of_all_services.py @bansalnitish @kevinlee12
-/scripts/create_topological_sort_of_all_services_test.py @bansalnitish @kevinlee12
-/scripts/docstrings_checker.py @kevinlee12
-/scripts/docstrings_checker_test.py @kevinlee12
-/scripts/install_chrome_on_travis.py @kevinlee12
-/scripts/install_chrome_on_travis_test.py @kevinlee12
-/scripts/install_prerequisites.sh @kevinlee12
-/scripts/install_third_party.py @kevinlee12
-/scripts/install_third_party_libs.py @kevinlee12
-/scripts/install_third_party_libs_test.py @kevinlee12
-/scripts/install_third_party_test.py @kevinlee12
-/scripts/pre_commit_hook.py @kevinlee12
-/scripts/pre_commit_hook_test.py @kevinlee12
-/scripts/pre_push_hook.py @kevinlee12
-/scripts/pre_push_hook_test.py @kevinlee12
-/scripts/release_scripts/deploy.py @ankita240796 @kevinlee12
-/scripts/release_scripts/deploy_test.py @ankita24079 @kevinlee12
-/scripts/run_backend_tests.py @kevinlee12
-/scripts/run_e2e_tests.py @kevinlee12
-/scripts/run_e2e_tests_test.py @kevinlee12
-/scripts/run_frontend_tests.py @kevinlee12
-/scripts/run_performance_tests.py @kevinlee12
-/scripts/run_presubmit_checks.py @kevinlee12
-/scripts/run_tests.py @kevinlee12
-/scripts/script_import_test.py @kevinlee12
-/scripts/concurrent_task_utils.py @kevinlee12 @Hudda
-/scripts/setup.py @kevinlee12
-/scripts/setup_gae.py @kevinlee12
-/scripts/setup_gae_test.py @kevinlee12
-/scripts/setup_test.py @kevinlee12
-/scripts/start.py @kevinlee12
-/scripts/third_party_size_check.py @kevinlee12
-/scripts/typescript_checks.py @ankita240796 @kevinlee12
-/scripts/typescript_checks_test.py @ankita240796 @kevinlee12
+/scripts/*.py @DubeySandeep
```

Comment:
I've done it this way so that any PR which adds new python files gets assigned to CODEOWNER and base on the requirement CODEOWNER can assign it to anyone else.

Reply:
So say for files like these `/scripts/create_topological_sort_of_all_services*.py` you'll be still the codeowner, right? 

I mean for some parts under scripts/*.py we have codeowners and on top of those you'll be codeowner as well. Right?

Reply:
> So say for files like these /scripts/create_topological_sort_of_all_services*.py you'll be still the codeowner, right?

Yep! In general, later lines override earlier ones. For details,	see the example at the bottom of this page:	https://help.github.com/en/articles/about-code-owners

> I mean for some parts under scripts/*.py we have codeowners and on top of those, you'll be codeowner as well. Right?

Yeah I'll be owning all the files (not dirs) and the files listed below this line will be owned by the users assigned in that line.